include/godot_cpp/classes/wrapped.hpp

```diff
@@ -96,11 +73,20 @@ struct Creator<T, typename std::enable_if<std::is_base_of_v<godot::Wrapped, T>>:
 
 #define GDCLASS(m_class, m_inherits)                                                                                                                                  \
 private:                                                                                                                                                              \
+	void operator=(const m_class &p_rval) {}                                                                                                     					  \
 	friend class ClassDB;                                                                                                                                             \
                                                                                                                                                                       \
 	using SelfType = m_class;                                                                                                                                         \
                                                                                                                                                                       \
 protected:                                                                                                                                                            \
+	virtual const char *_get_class() const override {																												  \
+		return get_class_static();																																	  \
+	}																																								  \
+																																									  \
+	virtual const GDNativeInstanceBindingCallbacks*_get_bindings_callbacks() const override {																		  \
+		return &___binding_callbacks;																																  \
+	}																																								  \
+																																									  \
```

Comment:
There seems to be a style issue here where the alignment of the trailing `\` is done with tabs instead of spaces.